seatunnel-connectors/plugin-flink-sink-doris/src/main/java/io/github/interestinglab/waterdrop/flink/sink/DorisSink.java

```diff
@@ -0,0 +1,108 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.interestinglab.waterdrop.flink.sink;
+
+import io.github.interestinglab.waterdrop.common.PropertiesUtil;
+import io.github.interestinglab.waterdrop.common.config.CheckConfigUtil;
+import io.github.interestinglab.waterdrop.common.config.CheckResult;
+import io.github.interestinglab.waterdrop.config.Config;
+import io.github.interestinglab.waterdrop.flink.FlinkEnvironment;
+import io.github.interestinglab.waterdrop.flink.batch.FlinkBatchSink;
+import io.github.interestinglab.waterdrop.flink.stream.FlinkStreamSink;
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.operators.DataSink;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.datastream.DataStreamSink;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.java.BatchTableEnvironment;
+import org.apache.flink.table.api.java.StreamTableEnvironment;
+import org.apache.flink.types.Row;
+
+import java.util.Properties;
+
+public class DorisSink implements FlinkStreamSink<Row, Row>, FlinkBatchSink<Row, Row> {
+
+    private Config config;
+    private String fenodes;
+    private String username;
+    private String password;
+    private String tableName;
+    private String dbName;
+    private int batchSize = 5000;
+    private long batchIntervalMs = 10;
+    private int maxRetries = 1;
+    private Properties streamLoadProp = new Properties();
+
+    @Override
+    public Config getConfig() {
+        return config;
+    }
+
+    @Override
+    public void setConfig(Config config) {
+        this.config = config;
+    }
+
+    @Override
+    public CheckResult checkConfig() {
+        return CheckConfigUtil.check(config, "fenodes", "username", "password", "table_name", "db_name");
+    }
+
+    @Override
+    public void prepare(FlinkEnvironment prepareEnv) {
+        fenodes = config.getString("fenodes");
+        username = config.getString("username");
+        tableName = config.getString("table_name");
+        password = config.getString("password");
+        dbName = config.getString("db_name");
+        if (config.hasPath("doris_sink_batch_size")) {
+            batchSize = config.getInt("doris_sink_batch_size");
+        }
+        if (config.hasPath("doris_sink_interval")) {
+            batchIntervalMs = config.getInt("doris_sink_interval");
+        }
+        if (config.hasPath("doris_sink_max_retries")) {
+            maxRetries = config.getInt("doris_sink_max_retries");
+        }
+
+        String producerPrefix = "doris_sink_properties.";
+        PropertiesUtil.setProperties(config, streamLoadProp, producerPrefix, false);
+    }
+
+    @Override
+    public DataSink<Row> outputBatch(FlinkEnvironment env, DataSet<Row> dataSet) {
+        batchIntervalMs = 0;
```

Comment:
Why set it to zero here?
And I think `try flush` every 1ms is unnecessary. how about 1 second or maybe longer?